app/auth/views.py

```diff
@@ -0,0 +1,100 @@
+""" authentication blueprint """
+
+import os
+import re
+import jwt
+import datetime
+from werkzeug.security import  check_password_hash, generate_password_hash
+from flask import Blueprint, jsonify, request
+from app import app
+from app.models import db, User
+
+
+auth = Blueprint('auth', __name__)
+
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS']=False
+app.config['SECRET_KEY']=os.getenv('SECRET_KEY')
+app.config['SQLALCHEMY_DATABASE_URI']=os.getenv('DATABASE_URL')
+
+
+@app.route('/api/v2/auth/register', methods=['POST'])
+def register_user():
```

Comment:
Looks like we already have this code here https://github.com/peterwade153/weconnect/pull/6/files#diff-1332faad1580b741aa03f276f41c7222R55 😕 
It looks kinda different but does the same thing.